app/controllers/ajax_controller.rb

```diff
@@ -56,6 +56,45 @@ def test
     render(:text => 'test')
   end
 
+
+  #Used by infite scroll
+  def index_rss_log
+    User.current = get_session_user
+    if !params[:type].blank?
+      types = params[:type].split & (["all"] + RssLog.all_types)
+      query = find_or_create_query(:RssLog, type: types.join(" "))
+    else
+      query = find_query(:RssLog)
+      query ||= create_query(:RssLog, :all,
+                             type: @user ? @user.default_rss_type : "all")
+    end
+
+    show_selected_rss_logs(query, id: params[:id].to_s, always_index: true)
+  end
+
+  #used by infinite scroll
+  def show_selected_rss_logs(query, args = {})
+    #store_query_in_session(query)
+    set_query_params(query)
+    args = {
+        matrix: true,
+        include: {
+            location: :user,
+            name: :user,
+            observation: [:location, :name, { thumb_image: :image_votes }, :user],
+            project: :user,
+            species_list: [:location, :user]
+        }
+    }.merge(args)
+
+    @user = get_session_user
+    @types = query.params[:type].to_s.split.sort
+    @links = []
+
+    show_index_of_objects(query, args)
+    render(template: "/observer/list_rss_logs", layout: nil)
```

Comment:
I think you can shorten this to 
`render("/observer/list_rss_logs", layout: nil)`
See http://guides.rubyonrails.org/layouts_and_rendering.html, 2.2.3 Rendering an Action's Template from Another Controller.